pyvista/plotting/widgets.py

```diff
@@ -1883,11 +1898,41 @@ def add_checkbox_button_widget(
         background_color : color_like, optional
             The background color of the button. Default is ``'white'``.
 
+        Examples
+        --------
+        The following example generates a static image of the widget.
+
+        >>> import pyvista as pv
+        >>> mesh = pv.Sphere()
+        >>> pl = pv.Plotter()
+        >>> actor = pl.add_mesh(mesh)
+        >>> def toggle_vis(flag):
+        ...     actor.SetVisibility(flag)
+        >>> pl.add_checkbox_button_widget(toggle_vis, value=True)
+        >>> pl.show()
+
+        Download the interactive example at :ref:`checkbox_widget_example`.
+
         Returns
         -------
         vtk.vtkButtonWidget
             The VTK button widget configured as a checkbox button.
 
+        Examples
+        -------
+        The following example generate a static image of the widget.
+
+        >>> import pyvista as pv
+        >>> mesh = pv.Sphere()
+        >>> p = pv.Plotter()
+        >>> actor = p.add_mesh(mesh)
+        >>> def toggle_vis(flag):
+        ...    actor.SetVisibility(flag)
+        >>> p.add_checkbox_button_widget(toggle_vis, value=True)
+        >>> p.show()
+
+        Download the interactive example at :ref:`checkbox_widget_example`.
+
```

Comment:
Actually, this block seems duplicated now. Was this meant to be placed into the docs of another method?

Reply:
I’m learning sphinx and merged requests as I go. Thank you for your patience.

Reply:
Thank _you_ for the contribution! :)